### DISCUSSION TITLE
Is the order of returned catalogItems always the same and has any kind of meaning?

### DISCUSSION THREAD

**Discussion by Schubi1981** [amzn/selling-partner-api-models#3369] [General] [answered]
When making a call to the catalogItem-API by e.g. an EAN it is possible to get a result with multiple catalogItems with different ASINs. Is the order of the catalogItems for such a result always the same? And is there any kind of meaning in the ordering? Are they ordered alphabetically by any property? Or is the first catalogItem in the list kind of "the best" result?

**Comment by chapmanjw** [comment, marked as answer]
@Schubi1981 When querying by ASIN, the results will be in the same order they were requested. When querying by identifier (EAN, UPC, etc.), the results will be sorted by ASIN. When querying by keyword (that has not been detected as an identifier), the results will be sorted by relevance.
